library/src/main/java/com/chuckerteam/chucker/internal/ui/error/ErrorActivity.kt

```diff
@@ -94,13 +86,21 @@ internal class ErrorActivity : BaseChuckerActivity() {
     }
 
     private fun populateUI(throwable: RecordedThrowable) {
-        title.text = throwable.formattedDate
-        tag.text = throwable.tag
-        clazz.text = throwable.clazz
-        message.text = throwable.message
-        stacktrace.text = throwable.content
+        errorBinding.apply {
```

Comment:
I don't agree that we should always stick to one particular scope function.
`with` or `apply` decision is based mainly on actions applied and on how you read it. 
In `onCreate()` we are doing something using `errorBinding`, that is why I used `with` there.
In `populateUI()` I used `apply`, because I explicitly mean that I want to apply some properties to items from this binding.

Reply:
It is still not clear to me to be honest. Should I split binding into two blocks of `with` and `apply` if I first do something with it and the apply some properties to it?

Reply:
No. I would stick to one function in case of using it in the same place. But in this particular case I use these extensions in different functions. 
So here it is just a matter of preference, because there are no clear conventions on these functions usage. Also, we don't care about the return type in both cases, since we are just removing multiple repetitions of `errorBinding` to improve readability. Finally, we will end up with the same Java code generated with additional variable generated, so there is no difference in preferring one or another in terms of some performance benefits.

Reply:
Ok, I got it now.